python_src/src/lamp_py/performance_manager/l1_rt_metrics.py

```diff
@@ -707,7 +757,6 @@ def process_metrics_table(
             == VehicleEventMetrics.trip_stop_hash,
             isouter=True,
         )
-        .where(all_new_metrics.c.travel_time_seconds > 0)
```

Comment:
so with this where clause, we were dropping out rows that had a `Null` travel time because this evaluated to False, causing us to loose headway info?

Reply:
Yes, it would drop the entire metrics records, if the travel time was negative or null.